Scripts/Services/Discovery/V1/Model/QueryAggregation.cs

```diff
@@ -24,7 +24,7 @@ namespace IBM.Watson.Discovery.V1.Model
     /// <summary>
     /// An aggregation produced by  Discovery to analyze the input provided.
     /// </summary>
-    [JsonConverter(typeof(JsonSubtypes), "Type")]
+    [JsonConverter(typeof(JsonSubtypes), "type")]
```

Comment:
I'm not so sure if I should include this change or not. I just added it here. Let me know if we need to convert it back

Reply:
correct, it should be lower case 👍